data/rulesets.js

```diff
@@ -11,24 +11,28 @@ exports.BattleFormats = {
 	standard: {
 		effectType: 'ValidatorRule',
 		name: 'Standard',
+		desc: ["The standard ruleset for all offical Smogon singles tiers (Ubers, OU, etc.)"],
 		ruleset: ['Sleep Clause Mod', 'Species Clause', 'Nickname Clause', 'OHKO Clause', 'Moody Clause', 'Evasion Moves Clause', 'Endless Battle Clause', 'HP Percentage Mod', 'Cancel Mod'],
 		banlist: ['Unreleased', 'Illegal'],
 	},
 	standardnext: {
 		effectType: 'ValidatorRule',
 		name: 'Standard NEXT',
+		desc: ["The standard ruleset for the NEXT mod"],
 		ruleset: ['Sleep Clause Mod', 'Species Clause', 'Nickname Clause', 'OHKO Clause', 'HP Percentage Mod', 'Cancel Mod'],
 		banlist: ['Illegal', 'Soul Dew'],
 	},
 	standardubers: {
 		effectType: 'ValidatorRule',
 		name: 'Standard Ubers',
+		desc: ["The standard ruleset for [Gen 5] Ubers"],
```

Comment:
`[Gen 5]`???

Reply:
Yeah, the only format that uses this list is `[Gen 5] Ubers`

Reply:
That makes sense... should probably get rid of it, then?

Reply:
Should that format just use `'Standard'` with `'Evasion Moves Clause'` as a removed rule, then, or did you have something else in mind?

Reply:
Yes, do that.